lib/cloud_watch.ex

```diff
@@ -55,10 +61,19 @@ defmodule CloudWatch do
     :ok
   end
 
-  defp configure(opts) do
-    opts = Keyword.merge(Application.get_env(:logger, CloudWatch, []), opts)
-    format = Logger.Formatter.compile(Keyword.get(opts, :format, @default_format))
+  defp configure(name, opts) do
+    env = Application.get_env(:logger, name, [])
+    opts = Keyword.merge(env, opts)
+    Application.put_env(:logger, name, opts)
+
+    # Main Logger configuration
```

Comment:
I feel the code speaks for itself so this comment is not required in my opinion.